pupil/public/javascripts/mtc-check.js

```diff
@@ -122,60 +124,86 @@
   // This is key, the user only has a few seconds to submit an answer or
   // the page will auto-submit (even if there is no input from the user)
   GOVUK.submitPageOnTimeout = function () {
-    var form = document.getElementById('js-question-form')
+    const form = document.getElementById('js-question-form')
+    const questionTimeLimit = GOVUK.getQuestionTimeLimit()
+    
     // Prefix setTimeout with `global` scope for IE8
     global.setTimeout(function () {
       // Copy the user input from the <span> into the <form>
       GOVUK.addInputToForm()
       form.submit()
-    }, (pageInterval * 1000))
+    }, (questionTimeLimit * 1000))
   }
 
   // Provide an indication to the user of how much time they have remaining.
-  GOVUK.updatePageTimer = function () {
-    // set the page timer to the countdown on page load
-    var pageTimer = document.getElementById('js-page-timer')
-    GOVUK.setTextContent(pageTimer, pageInterval.toString())
-
-    setInterval(function () {
-      var remaining = Math.ceil((deadline - Date.now()) / 1000)
-      if (remaining < 0) { remaining = 0 }
-      GOVUK.setTextContent(pageTimer, remaining.toString())
-    }, 100)
+  // GOVUK.updatePageTimer = function () {
```

Comment:
As explained in the PR's description, 'time remaining' text has been removed as per UX request (see story as well).